index.html

```diff
@@ -401,118 +308,81 @@ <h2>
       <pre class="idl">
         [SecureContext]
         partial interface Navigator {
-          [SameObject] readonly attribute WakeLock wakeLock;
+          [SameObject] readonly attribute ScreenWakeLock screenWakeLock;
```

Comment:
If we're making another breaking change why not move the `request()` method to this partial interface and call it `requestScreenWakeLock()`?

Reply:
That said I would prefer not to completely remove any hint of extensibility from this API. Why can't we leave the interface as it is except for removing the `WakeLockType` parameter? Extensions to this specification can define the default value to be `"screen"`. This was my suggestion in #253.

Reply:
I don't mind, tbh... I was going with @bholley's suggestion from https://github.com/mozilla/standards-positions/issues/210#issuecomment-597189110 of splitting them into `navigator.systemLock.` and `navigator.whateverLock.` and so on... what I kinda do like in the new approach is having a single identifying string everywhere. 

one nice thing about doing `navigator.whateverLock` is that it's easy to check if it's supported: `"whateverLock" in navigator`, instead of having to call `navigator.wakeLock.request()` and having to await a promise to reject with a `Not SupportedError` exception.

Reply:
> If we're making another breaking change why not move the request() method to this partial interface and call it requestScreenWakeLock()?

Not a bad option, actually :)

Reply:
Let's keep the existing API structure then and only remove references to dedicated workers and the "system" WakeLockType. This does not need to be a breaking change.

Reply:
Ok, sounds like a plan. I'll revert and send an update soon! Thanks for the feedback @reillyeon!

Reply:
@marcoscaceres do you think you could update this PR by this week?

I can check with others to help if you're busy to get this PR into a mergeable state (@kenchris is on vaca, so not bugging him). I'd like to get this PR landed since it's blocking progress on the split for which the group has consensus on.

Reply:
ok it...

Reply:
on it.. even!